src/thunks.js

```diff
@@ -39,36 +39,42 @@ export function makeWSConnection() {
   return (dispatch) => {
     const authToken = getAuthToken();
     const url = 'ws://' + window.location.host + '/api/timers/ws-connect/';
-    let wsConnection
-      = new WebSocket(url);
+    let intervalId;
+    let wsConnection = new WebSocket(url);
+
     wsConnection.onopen = (e) => {
       wsConnection.send(JSON.stringify({
         command: 'authenticate',
         token: authToken
       }));
     };
+
     wsConnection.onerror = (e) => {
```

Comment:
This would be called when a connection attempt fails right? Don't you need to retry here as well?